bspysmg/model/training.py

```diff
@@ -121,17 +139,17 @@ def generate_surrogate_model(
 
 
 def train_loop(
-    model,
-    info_dict,
-    dataloaders,
-    criterion,
-    optimizer,
-    epochs,
-    amplification,
-    start_epoch=0,
-    save_dir=None,
-    early_stopping=True,
-):
+    model : torch.nn.Module,
+    info_dict : dict,
+    dataloaders : list,
```

Comment:
what type have the elements of this list?

Reply:
Made the necessary change - List[torch.utils.data.DataLoader]